### PR TITLE
Remove deprecated method from README.

### PR DESCRIPTION
**Pull request by brettfishman**
Since it appears `Pwwka::Transmitter.send_message_safely` is a deprecated method, I removed reference to it in the README. I also tried to clarify the error handling strategies in lieu of this removed documentation.

Fixed some tiny typos, and proposed a small reformatting as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,16 +91,24 @@ Pwwka::Transmitter.send_message!(payload, routing_key)
 
 The payload should be a simple hash containing primitives. Don't send objects because the payload will be converted to JSON for sending.
 
-If an exception is raised sending your message, the bang form—`send_message!`—will pass that exception along to you and also blow up.  If
-you don't want that:
+Error handling can be configured by passing one of:
+
+```text
```
Comment (davetron5000) [inline README.md]: Should this be a bullet list?

File: README.md [hunk 2]
```diff
@@ -91,16 +91,24 @@ Pwwka::Transmitter.send_message!(payload, routing_key)
...
+Pwwka::Transmitter.send_message!(payload, routing_key, on_error: :ignore)
 ```
 
+
 #### Error Handling
 
-This method accepts several strategies for handling errors, pass in using the `on_error` parameter:
+This method accepts several strategies for handling errors, passed in using the `on_error` parameter:
 
   * `:raise`: Log the error and raise the exception received from Bunny. (default strategy)
```
Comment (davetron5000) [inline README.md]: Do we need this still if we are listing them out above?  Or can we omit the list above?

File: lib/pwwka/transmitter.rb [hunk 3]
```diff
@@ -42,8 +42,7 @@ def initialize
     # Returns true
     #
     # Raises any exception generated by the innerworkings of this library.
-    def self.send_message!(payload, routing_key, on_error: :raise,
-        delayed: false, delay_by: nil)
+    def self.send_message!(payload, routing_key, on_error: :raise, delayed: false, delay_by: nil)
```
Comment (davetron5000) [inline lib/pwwka/transmitter.rb]: Can we omit whitespace changes to keep the diff smaller?

File: README.md (hunk 2 above)
Comment (brettfishman) [inline README.md]: Yep, good call. I'll just omit what I added in the above section.

File: lib/pwwka/transmitter.rb (hunk 3 above)
Comment (brettfishman) [inline lib/pwwka/transmitter.rb]: Sure, I'll revert this.

**Comment by davetron5000** [comment]
Can we merge this?

**Comment by brettfishman** [comment]
Done.

**Pull request merged**